docs/home/whats-new-2-16.md

```diff
@@ -69,3 +69,16 @@ date: 2023-Dec-4
 #### OpenShift Support
 
 * Updated installation prerequisites. For more information, see [Kubernetes support matrix](../admin/runai-setup/cluster-setup/cluster-prerequisites.md#releases). -->
+
+## Deprecation Notifications
+
+Deprecation notifications allow you to plan for future changes in the Run:ai Platform. Deprecated features will be available for **two** versions ahead of the notification. For questions, see your Run:ai representative.
+
+### Project migration
+
+* Run:ai will be deprecating the migration of projects between departments. This affects:
+  
+    * API&mdash;the `departmentId` field will be marked as deprecated in the `projects` endpoint.
```

Comment:
@lavianalon
Sorry the comment system in VSCode erased the previous ones. Let's try this:
```suggestion
    * API&mdash;the `departmentId` field will be marked as deprecated in the`put` endpoint in the `projects` category.
```